web/src/html.js

```diff
@@ -255,24 +258,34 @@ async function buildClaimOgMetadata(uri, claim, overrideOptions = {}, referrerQu
     const videoUrl = generateEmbedUrlEncoded(claim.canonical_url);
     head += `<meta property="og:video" content="${videoUrl}" />`;
     head += `<meta property="og:video:secure_url" content="${videoUrl}" />`;
-    // type text/html since we use embeds
     head += `<meta property="og:video:type" content="text/html" />`;
     if (channel) {
       head += `<meta name="og:video:series" content="${channel}"/>`;
     }
-    head += `<meta name="twitter:card" content="player"/>`;
-    head += `<meta name="twitter:player" content="${videoUrl}" />`;
+    head += `<meta property="og:video:width" content="${mediaWidth}"/>`;
+    head += `<meta property="og:video:height" content="${mediaHeight}"/>`;
```

Comment:
This block handles `audio` too.  Does it make sense to add `video` stuff?

Reply:
I didn't change that part. I assume it's in there for the placeholder image...?